tests/test_hub.py

```diff
@@ -0,0 +1,53 @@
+from pathlib import Path
+
+import pytest
+from huggingface_hub.utils import LocalEntryNotFoundError
+
+from vllm_tgis_adapter.tgis_utils.hub import (
+    convert_files,
+    download_weights,
+    weight_files,
+    weight_hub_files,
+)
+
+
+def test_convert_files():
+    model_id = "bigscience/bloom-560m"
+    local_pt_files = download_weights(model_id, extension=".bin")
+    local_pt_files = [Path(p) for p in local_pt_files]
+    local_st_files = [
+        p.parent / f"{p.stem.removeprefix('pytorch_')}.safetensors"
+        for p in local_pt_files
+    ]
+    convert_files(local_pt_files, local_st_files, discard_names=[])
+
+    found_st_files = weight_files(model_id)
+
+    assert all(str(p) in found_st_files for p in local_st_files)
+
+
+def test_weight_hub_files():
```

Comment:
Marks should be added to other tests aswell.

Reply:
Since all of the tests in the file call for HF data I used the marker global variable on the module:

https://docs.pytest.org/en/7.1.x/reference/reference.html#globalvar-pytestmark

Reply:
Good call